src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/retry/MessageProducerImpl.java

```diff
@@ -24,9 +23,6 @@
 
     private CHKafkaProducer producer;
```

Comment:
My intellij is flaggin that these 3 fields, logger, avroSerializer and producer can all be marked as final